### ISSUE TITLE
AttributeError: 'module' object has no attribute 'logger'

### ISSUE THREAD

**Issue by ddib-ccde** [thombashi/tcconfig#46]
I'm failing to run tcconfig with the following error message:

```
sudo tcset --device eno1 --delay 50 --direction incoming
Traceback (most recent call last):
  File "/usr/local/bin/tcset", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/tcconfig/tcset.py", line 176, in main
    subprocrunner.logger.level = options.log_level
AttributeError: 'module' object has no attribute 'logger'
```

I have tcconfig working on an almost identical box from OS and Python perspective. This is the non working one:

```
uname -a
Linux Lab-desktop 4.4.0-59-generic #80-Ubuntu SMP Fri Jan 6 17:47:47 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

```
python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocrunner
>>> dir(subprocrunner)
['CommandNotFoundError', 'InvalidCommandError', 'SubprocessRunner', 'Which', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '__path__', '_error', '_logger', '_subprocess_runner', '_which', 'absolute_import', 'set_log_level', 'set_logger']
>>> 
```

This is the working one:

```
uname -a
Linux daniel-tc 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

```
python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocrunner
>>> dir(subprocrunner)
['CommandNotFoundError', 'InvalidCommandError', 'SubprocessRunner', 'Which', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '__path__', '_error', '_subprocess_runner', '_which', 'absolute_import', 'logger']
>>>
```

On the non working one it seems logger is missing from the dir(subprocrunner) command. 

Not sure if this is an issue with Python somehow or if something changed in tcconfig?

**Comment by ddib-ccde** [comment]
I was able to get this working by running the pre version.

`sudo pip install tcconfig --upgrade --pre`

```
python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocrunner
>>> dir(subprocrunner)
['CommandNotFoundError', 'InvalidCommandError', 'SubprocessRunner', 'Which', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '__path__', '_error', '_logger', '_subprocess_runner', '_which', 'absolute_import', 'set_log_level', 'set_logger']
>>> 
```

**Comment by thombashi** [comment]
Thank you for your report.

The cause of the problem is a dependency package version.
`subprocrunner` package interface is changed in `4.2` which breaks backward compatibility.
tcconfig 0.7.0-alpha version responds to this change while the latest release version of the tcconfig (0.6.6) is not.
Would you do one of the following workarounds?

1. Using `tcconfig 0.6.6` with `subprocrunner 0.4.5` (fixed backward compatibility)
    - `pip install subprocrunner --upgrade`
2. Using `tcconfig 0.6.6` with `subprocrunner 0.4.1`
    - `pip install subprocrunner==0.4.1`
3. Using `tcconfig 0.7.0-alphaN`

Sorry for the inconvenience.

**Comment by ddib-ccde** [comment]
Yes. I already have the pre installed and it works.

Thanks, @thombashi !

**Comment by thombashi** [comment]
@ddib-ccde 
I've released `tcconfig 0.7.0`.
I recommend to use this version 😃 .
